meepo/apps/eventsourcing/pub.py

```diff
@@ -92,8 +92,9 @@ def session_prepare(self, session, _):
             for obj in objs:
                 evt_name = "%s_%s" % (obj.__table__.fullname, action)
                 evt[evt_name].add(obj)
+                pks.add(self._pk(obj))
                 self.logger.debug("%s - session_prepare: %s -> %s" % (
-                    session.meepo_unique_id, evt_name, evt))
+                    session.meepo_unique_id, evt_name, pks))
```

Comment:
it seems there's a bug here. the pks will not be cleared during the loop? @maralla